File: AverageRandom.java
import java.util.Scanner;
public class AverageRandom{
	public static void main(String[] args){
		int n; int m = 0; int sum = 0; double average = 0.0; int above = 0;
		Scanner sc = new Scanner(System.in);
		
		System.out.print("n? ");
		n = sc.nextInt();
		
		int C[];
		C = new int[n];
		
		for(int i = 0; i<n; i++){
			C[i] = (int)(Math.random()*9)+1;
			System.out.print(C[i] + " ");
			sum = sum + C[i];
		}
		System.out.println("");
		average = ((double)sum/n);
		System.out.println("Average = " + average);
		
		for(int i = 0; i<n; i++){
			if(C[i]>average) above = above + 1;
		}
		
		System.out.println("Number of values above the average = " + above);
	}
}

File: 7th week array/AverageRandom.c
#include <stdio.h> 
#include <stdlib.h> 
#include<time.h> 

int main(void){ 
    
    srand(time(0));
    int i; int n; double sum = 0; double average; int above = 0;
  
  	printf("n? ");
  	scanf("%d",&n);
  	
  	int A[n];
  	
    for(i = 0; i<n; i++){
    	A[i] = rand()%9 + 1;
        printf("%d ", A[i]);
        sum = sum + A[i];
	}
	printf("\n");
	average = sum/n;
	printf("Average = %f\n", average);
	
	for(i = 0; i<n; i++){
        if(A[i]>average){
        	above += 1;
		}
	}
	
	printf("Number of values above the average = %d", above);
	
    return 0; 
} 
